Traceback (most recent call last):
  File "main.py", line 16, in <module>
    train_dataset, val_dataset = get_generator.get_dataset(config['data_gen'])
  File "/zhome/89/a/117273/Desktop/semi_active_seizure/src/data/get_generator.py", line 10, in get_dataset
    subjects_to_use = train)
  File "/zhome/89/a/117273/Desktop/semi_active_seizure/src/data/datagenerator.py", line 152, in __init__
    samples = self._prefetch_from_seg(samptemp)
  File "/zhome/89/a/117273/Desktop/semi_active_seizure/src/data/datagenerator.py", line 236, in _prefetch_from_seg
    sample = self._get_segment(item)
  File "/zhome/89/a/117273/Desktop/semi_active_seizure/src/data/datagenerator.py", line 204, in _get_segment
    sig = self.data_file[item['path']]
  File "/zhome/89/a/117273/.local/lib/python3.7/site-packages/dataapi/data_collection.py", line 45, in __getitem__
    item = super().__getitem__(name)
  File "h5py/_objects.pyx", line 54, in h5py._objects.with_phil.wrapper
  File "h5py/_objects.pyx", line 55, in h5py._objects.with_phil.wrapper
  File "/zhome/89/a/117273/miniconda3/envs/semi_active_seiz/lib/python3.7/site-packages/h5py/_hl/group.py", line 264, in __getitem__
    oid = h5o.open(self.id, self._e(name), lapl=self._lapl)
  File "h5py/_objects.pyx", line 54, in h5py._objects.with_phil.wrapper
  File "h5py/_objects.pyx", line 55, in h5py._objects.with_phil.wrapper
  File "h5py/h5o.pyx", line 190, in h5py.h5o.open
  File "h5py/h5i.pyx", line 40, in h5py.h5i.wrap_identifier
  File "<frozen importlib._bootstrap>", line 416, in parent
KeyboardInterrupt
